src/lib/frontend/d_cnf.ml

```diff
@@ -42,6 +42,11 @@ module DT = DE.Ty
 module Id = DStd.Id
 module B = DStd.Builtin
 
+let unsupported msg =
+  Format.kasprintf
+    (fun str -> Errors.(run_error (Unsupported_feature str)))
+    msg
```

Comment:
I guess we can use `Fmt.kstr` here :)? Besides we can move the function in `Utils`.

I'm not sure `unsupported_feature` is the appropriate error to produce in all these cases.

Reply:
`Errors` depends on `Util` unfortunately, but I can use `Fmt`